radar.csv

```diff
@@ -31,3 +31,4 @@ Edge,adopt,platforms,true,"Edge is Microsoft's evergreen browser. We support it
 IE 11,adopt,platforms,true,"IE 11 is the default browser on Windows 10 and makes up approximately 10% of the visits to Redgate.com. We support it by default"
 IE 10 and under,retire,platforms,true,"We do not support IE10 and under"
 Raven DB,retire,tools,true,"We previously used Raven DB in DLM Dashboard but our experience was not positive."
+SQL Change Automation,explore,tools,true,"Redgate SQL Change Automation is our commerical offering to help people manage the state of databases as part of application deployment. We should investigate if/how to use our own tools as part of our development activities."
```

Comment:
Hate to be the broken record about this but this has a direct effect to our choice around this as a team. So, I want to make sure that I understand this :) 

Which team is going to explore this and when the results are going to be made available?

Reply:
Honestly, I've no idea. That's not my call to make :) @fffej?

Reply:
👍 

> FWIW, I don't believe something being in _Explore_ should dictate behaviour. This is showing our desire or intent to explore something, not a hard commitment to it.

I am not saying this is wrong but I have the opposite interpretation of this ring. Should we first make this ring clear and decide whether this should be added here or not? I am not attached to which way we go as well, as long as we make it clear. My belief is that doing some would make it easier to interpret the radar accurately.

Reply:
> If there is a suitable thing to _Adopt_ instead, and we have work to be done, that should still be the default choice?

SQL Clone and Data Catalog have been using EF Core successfully. As far as I can see, DLM Dashboard is about to do the same: https://github.com/red-gate/sqllighthouse/blob/kill-raven-master/source/RedGate.SQLLighthouse.SqlServerStorage/packages.config#L5 not sure what Monitor uses though. Also, I am not aware what CoreDEV is using, too.

Reply:
Definitely agree we should resolve #43. I'll mark this as blocked until then.

EF Core does seem to be our default at the moment.

(AFAIK SQL Monitor has a bespoke migrations-like thing. No idea on CoreDEV.)

Reply:
COREDev use SQL Compare pretty much manually for all of their processes. They use it to create/update a scripts folder, which is then committed to source control. And they use it to manually create change scripts which get passed around in Slack and run manually whenever an app deployment comes with a corresponding db change. It's all sounds a bit meh, but works well enough in practice that it's by no means their biggest pain point to solve.